src/bitfield.rs

```diff
@@ -240,6 +240,24 @@ impl<N: Unsigned + Clone> Bitfield<Variable<N>> {
     pub fn is_subset(&self, other: &Self) -> bool {
         self.difference(other).is_zero()
     }
+
+    /// Returns a new BitList of length M, with the same bits set as `self`.
+    pub fn extend<M: Unsigned + Clone>(&self) -> Result<Bitfield<Variable<M>>, Error> {
```

Comment:
I wonder if we should call this `resize` rather than `extend` because `extend` usually means "add more elements to this collection", as in:

https://doc.rust-lang.org/std/iter/trait.Extend.html

Reply:
renamed to resize